__tests__/test.js

```diff
@@ -45,3 +105,79 @@ describe('getOrThrow', () => {
         expect(getOrThrow('TIMEZONE')).to.equal(process.env.TIMEZONE);
     });
 });
+
+describe('createIdObject', () => {
+    it('should create a object id', () => {
+        const returnObject = createIdObject(
+            ['mockedUser', 'mockedUser@user.com'],
+            mockedContributed
+        );
+
+        expect(returnObject.alternateId).to.equal('mockedUser@user.com');
+        expect(returnObject.github).to.equal('mockedUser');
+
+        const contributionsObject = [
+            { id: '1', type: 'IssueCommentEvent' },
+            { id: '2', type: 'PullRequestEvent' },
+        ];
+
+        contributionsObject.forEach((contribution, index) => {
+            expect(returnObject.contributions[index].id).to.equal(contribution.id);
+            expect(returnObject.contributions[index].type).to.equal(contribution.type);
+        });
+    });
+});
+
+describe('parseDatesFromArgv', () => {
+    beforeEach(() => {
+        process.env = {
+            GITHUB_TOKEN: 'mockToken',
+            TIMEZONE: 'America/Los_Angeles',
```

Comment:
Will this cause the results to differ based on the time of year? (PST vs PDT)

Reply:
That's a really interesting point. I think the results will differ. But I do think that it's useful to test events that happen at `23:59:59 GMT-0800` to make sure Starfish is handling them correctly.
So, perhaps one way to handle this is to check to see which part of the year it is (PST or PDT) and write a test for each of those possibilities?

Reply:
Upon further reflection, I don't think this is a problem. 

The starfish code never looks at the current time, so it shouldn't matter whether the tests are being run during PST or PDT. What matters is the hard-coded dates (lines 142-143), which are the only dates that starfish will apply the TIMEZONE setting to. If those were to change between dates that were/will be PST to dates that were/will be PDT, then the test results could change. But as long as they don't change, I think using the US/LA time zone should be OK. 

I have been burned by time zones in tests before, so I was a bit overly cautious here. I should have thought it through before commenting.